scripts/build_network.sh

```diff
@@ -85,19 +85,28 @@ cleanup() {
     rm -f local.*.tar.gz
 }
 validate_arguments() {
-    # The script expects 3 arguments:
-    # 1. The path to a network config toml
-    # 2. The path to the directory containing the Namada binaries
-    # 3. The BASE_DIR of the chain
+    # The script expects 2 arguments:
+    # 1. The path to the directory containing the Namada binaries
+    # 2. The BASE_DIR of the chain
     
-    if [ "$#" -gt 3 ] || [ "$#" -lt 2 ]; then
-        echo "Error: Invalid number of arguments. Expected 2 or 3 arguments."
+    if [ "$#" -gt 2 ] || [ "$#" -lt 1 ]; then
+        echo "Error: Invalid number of arguments. Expected 1 or 2 arguments."
         echo "See the help page by running --help for more information."
         exit 1
     fi
 
-    NETWORK_CONFIG_PATH=$1
-    NAMADA_BIN_DIR=$2
+    SCRIPT_DIR="$(dirname $0)"
+    NETWORK_CONFIG_PATH="$SCRIPT_DIR/../genesis/e2e-tests-single-node.toml"
+
+    echo "Would you like to provide a custom genesis file? (y/n)"
+    read -r answer
+    if [ "$answer" = "y" ]; then
+        echo "Please provide the path to the genesis file:"
+        read -r genesis_path
+        NETWORK_CONFIG_PATH="$genesis_path"
+    fi
+    
+    NAMADA_BIN_DIR=$1
```

Comment:
btw since we're changing this script anyway, this variable should be updated
```suggestion
    NAMADA_BIN_DIR="$1"
```
the assignment wouldn't work properly if the path contained spaces